### ISSUE TITLE
Training on custom data

### ISSUE THREAD

**Issue by MvWouden** [redrock303/NeRFLiX_CPVR2023#5]
Hi, 

First of all, I want to say that the demo results look really promising.
I was wondering whether it was possible to train the model on custom data.
If so, what would be the procedure to get the data in the correct format, and what commands would I need to use?

**Comment by redrock303** [comment]
Thanks for your attention. You should collect paired training data, e.g., high-quality custom data from the following steps: (1) for each of nerf input views,  perform our view selection scheme to obtain the two most relevant views as reference views, (2) utilize our nds to degrade the target image, (3) repeat (1-2) to generate multiple paired training samples. (4) take advantage of these simulated samples to fine-tune our model. Also according to our latest work nerflix++, using part of real-rendered custom data (produced by your Nerf model ) for training/finetuning will be even better.
